nixos/modules/services/networking/shellhub-agent.nix

```
@@ -74,9 +89,11 @@ in {
         "time-sync.target"
       ];
 
-      environment.SERVER_ADDRESS = cfg.server;
-      environment.PRIVATE_KEY = cfg.privateKey;
-      environment.TENANT_ID = cfg.tenantId;
+      environment.SHELLHUB_SERVER_ADDRESS = cfg.server;
+      environment.SHELLHUB_PRIVATE_KEY = cfg.privateKey;
+      environment.SHELLHUB_TENANT_ID = cfg.tenantId;
+      environment.SHELLHUB_PREFERRED_HOSTNAME = cfg.preferredHostname;
+      environment.SHELLHUB_KEEPALIVE_INTERVAL = toString cfg.keepAliveInterval;
```

```suggestion
      environment = {
        SHELLHUB_SERVER_ADDRESS = cfg.server;
        SHELLHUB_PRIVATE_KEY = cfg.privateKey;
        SHELLHUB_TENANT_ID = cfg.tenantId;
        SHELLHUB_PREFERRED_HOSTNAME = cfg.preferredHostname;
        SHELLHUB_KEEPALIVE_INTERVAL = toString cfg.keepAliveInterval;
      };
```
We could reduce repetition of environment if you like.

---

Sure. I'll do it here and send the update.